templates/registration/login.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Hi</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login'%}" method="post">
+    {% csrf_token %}
+    <label>Username:
+      <input type="text" name="usernamne">

```

Comment:
There is a typo in the input field name attribute. It should be `name="username"` instead of `name="usernamne"`. This typo will prevent the form from correctly capturing the username input.